packages/store-engine/README.md

```diff
@@ -0,0 +1,159 @@
+# Wire
+
+This repository is part of the source code of Wire. You can find more information at [wire.com](https://wire.com) or by contacting opensource@wire.com.
+
+You can find the published source code at [github.com/wireapp](https://github.com/wireapp).
+
+For licensing information, see the attached LICENSE file and the list of third-party licenses at [wire.com/legal/licenses/](https://wire.com/legal/licenses/).
+
+## Store Engine
+
+Provider for the following storage engines: File, IndexedDB, Memory & LocalStorage.
+
+### Motivation
+
+#### One API to rule them all!
+
+![big deal](https://user-images.githubusercontent.com/469989/28491995-c5f0ea34-6efa-11e7-97d1-2f8b1d159981.jpg)
+
+Nowadays there are more and more storage possibilities and developers must be familiar with the characteristics of each individual solution to reliably store data. Because it can be sometimes hard to keep up with the highly dynamic world of data storages, we have developed a system which unifies the usage of [IndexedDB](https://developer.mozilla.org/docs/IndexedDB), [In-memory storage](https://en.wikipedia.org/wiki/In-memory_database), [File-based storage](https://nodejs.org/api/fs.html) and [LocalStorage](https://developer.mozilla.org/docs/Web/API/Window/localStorage). In addition we built some functionality (like a transient store which deletes data after a [TTL](https://en.wikipedia.org/wiki/Time_to_live)) on top.
+
+### Overview
+
+|       Engine       | Available in Browser | Available in Node.js |                                                        Description                                                         |
+| :----------------: | :------------------: | :------------------: | :------------------------------------------------------------------------------------------------------------------------: |
```

Comment:
I think the checkboxes is OK to have them centered.. but not the rest.

Reply:
Prettier messed up with my whole table:

![unbenannt](https://user-images.githubusercontent.com/469989/33261348-cff8782c-d362-11e7-9cbc-841b1651ed61.PNG)

It was much easier to edit without it:

![unbenannt](https://user-images.githubusercontent.com/469989/33261406-f97c3d14-d362-11e7-8f25-70373e606b9e.PNG)

Reply:
Then just add `<!-- prettier-ignore -->` in front of it..  and you are done :)

Reply:
👍